TestCases/ddes/flatplate/ddes_flatplate.cfg

```diff
@@ -226,4 +227,4 @@ SURFACE_ADJ_FILENAME= surface_adjoint
 OUTPUT_WRT_FREQ= 1000
 %
 % Screen output
-SCREEN_OUTPUT= (TIME_ITER, INNER_ITER, RMS_DENSITY, RMS_NU_TILDE, LIFT, DRAG)
```

Comment:
I removed the TIME_ITER to help with the regression scripts.   Though the "SIM_VALUES" that are used in the regression don't seem to be the final simulation residual/coefficient values.

Reply:
That may not be ideal, without time iter the script will look at time step 0, we should test more than that.
With time iter it will look at the first inner iteration for the timestep you want.